contracts/mocks/TestStaking.sol

```diff
@@ -1,22 +1,20 @@
 //SPDX-License-Identifier: MIT
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.20;
 
 import "erc721a/contracts/ERC721A.sol";
 import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 
 contract TestStaking {
     using EnumerableSet for EnumerableSet.UintSet;
+
     IERC721A private _nft;
     mapping(address => EnumerableSet.UintSet) private _stakers;
 
     constructor(address nft) {
         _nft = IERC721A(nft);
     }
 
-    function isStaked(
-        address staker,
-        uint256 tokenId
-    ) public view returns (bool) {
+    function isStaked(address staker, uint256 tokenId) public view returns (bool) {
```

Comment:
Is there a use case for wanting to know if a token is staked without having to pass in the owner's address?

Reply:
idk the context of this test, its an old one that was formatted by forge